Feature/monthly billing page

SlavaNekrish opened the pull request:
Monthly billing page content implementation.
Includes:

fetching data
component approach when organizing code
using amCharts4
create helpers and constants
styling using scss;
SFC

![image](https://github.com/user-attachments/assets/2b1ce4af-31b3-41e7-bf50-f4befe4ae17b)
![image](https://github.com/user-attachments/assets/4dba70a4-1c01-403b-8935-8935d154c97c)

SlavaNekrish committed:
add fetch functions to api folder

SlavaNekrish committed:
add chart to chartContainer

SlavaNekrish committed:
Merge branch 'master' into feature/monthly-billing-page

SlavaNekrish committed:
implement infopanels and first chart final shape

SlavaNekrish committed:
created smrtphones chart

iv-thomson reviewed (approved):
looks good
Where did you get the vite.timestamp file though?

SlavaNekrish committed:
delete cache file

SlavaNekrish commented:
I don’t know what file it is. It may have been created accidentally during the build of the project. In any case, I deleted this file and cleaned the cache
